apps/src/music/views/SoundsPanel2.tsx

```diff
@@ -252,7 +252,12 @@ const SoundsPanel: React.FunctionComponent<SoundsPanelProps> = ({
   const currentSoundRef: React.MutableRefObject<HTMLDivElement | null> =
     useRef(null);
 
-  const isDefaultSoundSelected = currentValue === library.getDefaultSound();
+  // Capture whether the initially-selected sound is the default sound, in which
+  // case we will do a smooth scroll into view, rather than scroll directly
+  // to the current sound.
+  const isDefaultSoundSelected = useRef(
```

Comment:
To check my understanding: we're using `useRef` so that the below `useEffect` only runs on initial render, and not whenever `currentValue` changes, right? If so, mind updating the comment to capture that?

Reply:
Yes, that's right.